test/export/test_hop.py

```diff
@@ -85,6 +85,7 @@ def forward(self, *args):
             kwargs = inp.kwargs
             ep = export(model, args, kwargs, strict=True)
             self._compare(model, ep, args, kwargs)
+        torch._dynamo.reset()
```

Comment:
Probably a better way is to do torch._dynamo.reset() in setUp()?

Reply:
teardown() would be more close to what we have now.

Reply:
Yeah, the cuda memory check appears to be a context manager: https://github.com/pytorch/pytorch/blob/main/torch/testing/_internal/common_utils.py#L3110-L3121. But we still don't know why we're getting a memory leak.. Need look into it more